generators/chipyard/src/main/scala/ConfigFragments.scala

```diff
@@ -105,7 +98,8 @@ class WithMultiRoCCHwacha(harts: Int*) extends Config(
     case MultiRoCCKey => {
       up(MultiRoCCKey, site) ++ harts.distinct.map{ i =>
         (i -> Seq((p: Parameters) => {
-          LazyModule(new Hwacha()(p)).suggestName("hwacha")
+          val hwacha = LazyModule(new Hwacha()(p)).suggestName("hwacha")
```

Comment:
What is the purpose of explicitly creating this val?

Reply:
Making it a `val` makes the `suggestName` redundant, but this was an artifact from testing. I'll remove this.